docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -334,18 +334,143 @@ In the NHANES example, the full first sentence of the above quote was:
+we should:
+
+- Name the gender- and sex-related factors our result pertains to, if clear.
+  If unclear (which is most of the time), call out the difficulty of disentangling factors
+  from each other, to prevent readers from jumping to the conclusion that it's always
+  biological factors that are important.
+- Acknowledge data limitations and proxy measures used.
+- Call for better data to be collected.
+
+Let's consider the example of reporting a result about pregnancy.
+In this case, while gender-related factors could play a role in e.g. pregnancy outcomes,
+the sex- or gender-related factor with the most direct relevance is clearly the ability
+to become pregnant.
+In this case, we might say:
+
+.. pull-quote::
+
+  In health research on pregnancy, it is important to identify the group
+  of people who can become pregnant.
+  Typically, a proxy measure of "women of reproductive age" is constructed for
+  this purpose, combining some indicator of female sex or gender with an age restriction
+  such as 15-49 years.
+  This measure does not account for infertility, for example by including people who
+  have had hysterectomies.
+  Additionally, when it is defined in a way that conflates sex with gender,
+  it will be highly inaccurate among transgender people and other gender minority populations.
+
+  To be more precise and inclusive in our language, we define our population of interest as
+  women and birthing people of reproductive age (WBPRA): people ages 15-49 who were assigned
+  female sex at birth.
+  Note that this definition retains the limitation about infertility.
+
+  Sex assigned at birth was not measured in any of our data sources,
+  so we used proxy measures for this attribute.
+  Notably, <source> and <source> only provided undifferentiated sex/gender measures,
+  making it unclear how our conclusions generalize to transgender people and other gender minority populations.
+  Due to these issues, our results are among WBPRA according to imperfect proxy measures.
+  To address these limitations, data would need to be collected using more robust measures
+  of sex- and gender-related factors. [cite guidelines for collecting these]
+
+On the other hand, if we were doing descriptive epidemiology and didn't have even an educated
+guess of what specific sex- and gender-related factors could be associated with our outcome:
+
+.. pull-quote::
+
+  Sex and gender are multi-dimensional concepts.
+  Collectively, they span a number of biological (sex-related) and sociocultural
+  (gender-related) factors which are highly correlated and causally entangled with one other. [citations!]
+  Frequently in health research, differences are attributed wholly to sex-related factors, when
+  in fact gender-related factors also play a part in those differences. [citation]
+  Since prevalence of <outcome> could be influenced by sex- and/or gender-related factors,
+  an ideal analysis would report prevalence by multiple of these factors.
+
```

Comment:
I think having an example somewhere - not sure if here is the right spot - about gender casual pathways might be helpful for readers. Things I think about in CVD: is one gender more likely to go to the doctor sooner, is one gender's pain taken more seriously in the emergency room. These are gender things but its easy to imagine how they would affect outcomes.